src/main/resources/application-dev.yml

```
@@ -0,0 +1,31 @@
+        use_sql_comments: true
+        highlight_sql: true
+        generate_statistics: true
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+    show-sql: true
+
+logging:
+  level:
+    org:
+      hibernate:
+        sql: debug
+        type.descriptor.sql: trace
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)